_data/projects/backstage-hacktoberfest.yml

```
@@ -0,0 +1,11 @@
+name: Backstage
+desc: Backstage is a web app built to support an NFT marketplace for Hacktoberfest 2022. 
+site: https://github.com/aryamankha/backstage-hacktoberfest
+tags:
+  - reactjs
+  - javascript
+  - tailwind
+  - nft
+upforgrabs:
+  name: up-for-grabs
+  link: https://github.com/aryamankha/backstage-hacktoberfest/issues
```

:art: not sure why this didn't get flagged - this link should point to the issues for the label above:

```suggestion
  link: https://github.com/aryamankha/backstage-hacktoberfest/labels/up-for-grabs
```